docs/en/latest/FAQ.md

```diff
@@ -424,3 +424,45 @@ $ curl http://127.0.0.1:9080/ip -i
 HTTP/1.1 200 OK
 ...
 ```
+
+## What is the `X-API-KEY` of Admin API? Can it be modified?
+
+1. The `X-API-KEY` of Admin API refers to the `apisix.admin_key.key` in the `config.yaml` file, and the default value is `edd1c9f034335f136f87ad84b625c8f1`. It is the access token of the Admin API.
```

Comment:
We may add a note that it's recommended to not use the default value for the sake of security.

Reply:
Updated.